app/assets/stylesheets/modules/gem.css

```diff
@@ -31,6 +31,11 @@
       overflow: auto;
       border-bottom: 1px solid #c1c4ca; } }
 
+.owners{
```

Comment:
I have fixed the positioning of curly brackets. If that is not what you meant, please elaborate :)